lte/gateway/c/core/oai/tasks/ngap/ngap_state_manager.cpp

```diff
@@ -93,6 +97,64 @@ void NgapStateManager::create_state() {
   create_ngap_imsi_map();
 }
 
+status_code_e NgapStateManager::read_state_from_db() {
+#if !MME_UNIT_TEST
+  /* Data store is Redis db. In this case actual call is made to Redis db */
+  StateManager::read_state_from_db();
+#else
+  /* Data store is a map defined in NGAPClientServicer.In this case call is NOT
+   * made to Redis db */
+  if (persist_state_enabled) {
```

Comment:
what is the purpose of this flag? looking at the code it seems like this else case if used for unit test scenario, what is the purpose to read this flag since anyways this will not readi/write to redis?

Reply:
Removed the flag, addressed review comment.